packages/blocks/shuffle/src/components/item.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FunctionComponent, useState } from "react";
+import DeleteIcon from "@mui/icons-material/Delete";
+import AddIcon from "@mui/icons-material/Add";
+import DragIndicatorIcon from "@mui/icons-material/DragIndicator";
+import { Box, IconButton, ListItem, TextField } from "@mui/material";
+import { Draggable } from "react-beautiful-dnd";
```

Comment:
Loading the block into the HASH app and dragging any item causes a crash:

```
Invariant failed: Cannot find droppable entry with id [droppable]👷‍ This is a development only message. It will be removed in production builds.
```
You can load it by running (from your block folder) `yarn dev`, and then in a HASH page click on the context menu and paste in the URL – http://localhost:63212/ in this case

Reply:
I've spent a long time trying to figure out a way to fix this to no avail. Thought it might be cause by Next's SSR, but I tried [this](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/api/reset-server-context.md) and it didn't seem to do the trick. I figure it must be a problem with rendering the blocks through portals that's causing the droppable containers' reference to change and that causes the crash.

I've now tried several other libraries to replace `react-beautiful-dnd`, the most promising being [dnd-kit](https://www.npmjs.com/package/@dnd-kit/core), which I haven't adopted yet because it increases the size of the block by dragging an element around. If I can't find a workaround for that, I'll probably have to go with [react-dnd](https://www.google.com/search?q=react-dnd&rlz=1C1CHBF_pt-PTPT1009PT1009&oq=react-dnd&aqs=chrome..69i57j35i39l2j0i512j69i65j69i60l3.2417j0j7&sourceid=chrome&ie=UTF-8), which is very flexible but brings no animation logic.

Reply:
I had a quick look at this and couldn't figure it out either. There are a tons of warnings in the console that might cause an issue even if we fixed the crash, so moving on to another library seems sensible.